src/Service/CoverStore/CloudinaryTransformationService.php

```diff
@@ -37,7 +37,7 @@ public function __construct(array $cloudinaryTransformations)
     /**
      * {@inheritdoc}
      */
-    public function transform(string $url, string $format = 'default'): string
+    public function transform(string $url, int $width, int $height, string $format = 'default'): ?string
```

Comment:
Why pass width, only height is used?

Reply:
Mainly to have it there if we need it later on...